cli/util.go

```diff
@@ -25,21 +26,37 @@ func NewVMTemplate(c *vmLauncher.ComposeTemplate) vmLauncher.ComposeTemplate {
 		panic(err)
 	}
 
+	globalNamespace := c.Namespace
```

Comment:
Given the fact that this single user's scope namespace assignment nature, I'd name this as `localNamespace` or  maybe use the `defaultNamespace` that you're using on line 31.

Reply:
I'm sorry, I almost forget it, could you please add this new option to the `README.md` file in the `create` section.

Reply:
Sure! My logic was that since it was the namespace for all VMs in the compose file it should be called global but you're right that default or local are better naming options.